### PR TITLE
minor fixes for data returned by api config_templates controller

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: app/views/api/v1/config_templates/show.json.rabl [hunk 1]
```diff
@@ -1,11 +1,4 @@
...
 attributes :name, :template, :snippet, :audit_comment, :id
-
-node do |template|
-  unless template.snippet?
-    child :template_kind do
-      attributes :name, :id
-    end
-  end
-end
+attributes :template_kind_id, :operatingsystem_ids, :unless => lambda {|t| t.snippet?}
```
Comment (ohadlevy) [inline app/views/api/v1/config_templates/show.json.rabl:4]: this breaks older rabl versions, and afair, latest rabl gem had another bug which we could not upgrade to, @isratrade ?

Comment (isratrade) [inline app/views/api/v1/config_templates/show.json.rabl:4]: @ohadlevy, @witlessbird .  We were using Rabl 0.7.6 which didn't have condition :if, :unless.  This was released in version 0.7.7 (on Nov 26), but there was an issue with the root json object randomly not appearing.  I don't see the root addressed in the change log.  It it's fixed in the latest version 0.7.9, then we can upgrade the gem and the code.

Comment (ohadlevy) [inline app/views/api/v1/config_templates/show.json.rabl:4]: @witlessbird also, afair we tried to keep the same json output in api V1, any changes to that needs to go to V2

Comment (dmitri-d) [inline app/views/api/v1/config_templates/show.json.rabl:4]: closing this one. i think it's worth it to simplify how template kinds are being handled, but we'll have to do it in v2 (if we'll do it)

**Pull request closed without merging**